website_product_document_download_counter/readme/USAGE.md

```
@@ -0,0 +1,9 @@
+Activate the "Count downloads" switch on a product document kanban tile. This checkbox is shown only if the "Publish on website" switch is already activated. A download counter will be displayed on the kanban tile.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)